.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       DSD_DOCKER_REGISTRY: "registry.service.dsd.io"
       DSD_DOCKER_IMAGE: "fala"
       ECR_DOCKER_REGISTRY: "926803513772.dkr.ecr.eu-west-1.amazonaws.com"
-      ECR_DOCKER_IMAGE: "laa-get-access/laa-fala"
+      ECR_DOCKER_IMAGE: "laa-get-access/laa-fala-webapp"
```

Comment:
_question_ What do you think about using `-web` instead of `-webapp`?

Reply:
I'm not bothered either way. I've created the ECR repos so will require a pull request in cloud platform environments repo, plus a change to the environment variables.